README.md

```diff
@@ -1,55 +1,94 @@
-# personnummer [![Build Status](https://travis-ci.org/personnummer/php.svg?branch=master)](https://travis-ci.org/personnummer/php) ![Packagist Version](https://img.shields.io/packagist/v/frozzare/personnummer) ![PHP from Packagist](https://img.shields.io/packagist/php-v/frozzare/personnummer)
+# personnummer [![Build Status](https://travis-ci.org/personnummer/php.svg?branch=master)](https://travis-ci.org/personnummer/php) ![Packagist Version](https://img.shields.io/packagist/v/personnummer/personnummer) ![PHP from Packagist](https://img.shields.io/packagist/php-v/personnummer/personnummer)
 
 Validate Swedish social security numbers.
 
 ## Installation
 
 ```
-composer require frozzare/personnummer
+composer require personnummer/personnummer
 ```
 
+## Methods
+#### Static
+| Method | Arguments                                                      | Returns  |
+| -------|:---------------------------------------------------------------|---------:|
+| parse  | string personnummer, [ array options<sup>[*](#options)</sup> ] | Instance |
+| valid  | string personnummer, [ array options<sup>[*](#options)</sup> ] | bool     |
+
+#### Instance
+| Method               | Arguments       | Returns |
+| ---------------------|:----------------|--------:|
+| format               | bool longFormat | string  |
+| getAge               | none            | int     |
```

Comment:
`getAge` will be removed in v3 and replaced with `age` property

Reply:
Seeing php only have properties in form of the magic `__get` method or through public methods, I think that its okay to have both a `getAge` and a magic `__get` for the properties.

I'm fine with it staying, whilst I wouldn't be angry with it gone :P

Reply:
I agree with @Johannestegner. No need to remove it for now.

Reply:
I thought it was the old `getAge` method and `age` is missing from the property list :)

Reply:
Ahh, okay, I missed that one. Could you add that @rasmusbe?